geodepy/tests/compare_to_webpage.py

```diff
@@ -0,0 +1,217 @@
+
+    for row_number, (lat1, lon1, lat2, lon2) in enumerate(coord_pairs):
+        # inverse vincentys
+        print(("Testing vincentys_inverse({}, {}, {}, {})"
+               " on row {}").format(lat1, lon1, lat2, lon2, str(row_number)))
+
+        (web_inv_fa_deg, web_inv_fa_min, web_inv_fa_sec,
+         web_inv_ra_deg, web_inv_ra_min, web_inv_ra_sec,
+         web_inv_ell_dist) = query_vincenty_inverse(*split_dms(lat1), *split_dms(lon1),
+                                                    *split_dms(lat2), *split_dms(lon2))
+        web_inv_az1to2, web_inv_az2to1 = (join_dms(web_inv_fa_deg, web_inv_fa_min, web_inv_fa_sec),
+                                          join_dms(web_inv_ra_deg, web_inv_ra_min, web_inv_ra_sec))
+
+        inv_ell_dist, inv_az1to2, inv_az2to1 = vincinv(dms2dd(lat1), dms2dd(lon1), dms2dd(lat2), dms2dd(lon2))
+
+        np.testing.assert_almost_equal((inv_ell_dist, dd2dms(inv_az1to2), dd2dms(inv_az2to1)),
+                                       (float(web_inv_ell_dist), float(web_inv_az1to2), float(web_inv_az2to1)),
+                                       decimal=1,
+                                       err_msg=("vincentys_inverse({}, {}, {}, {})"
+                                                " on row {}").format(lat1, lon1, lat2, lon2, str(row_number)))
+
+        # direct vincentys
+        print("Testing vincentys_direct({}, {}, {}, {}) on row {}".format(lat1,
+                                                                          lon1,
+                                                                          dd2dms(inv_az1to2),
+                                                                          dd2dms(inv_ell_dist),
+                                                                          str(row_number)))
+        (lat2_deg, lat2_min, lat2_sec,
+         lon2_deg, lon2_min, lon2_sec,
+         web_dir_ra_deg, web_dir_ra_min, web_dir_ra_sec) = query_vincenty_direct(*split_dms(lat1), *split_dms(lon1),
+                                                                                 web_inv_fa_deg, web_inv_fa_min,
+                                                                                 web_inv_fa_sec,
+                                                                                 web_inv_ell_dist)
+        web_dir_lat2, web_dir_lon2, web_dir_az2to1 = (join_dms(lat2_deg, lat2_min, lat2_sec),
+                                                      join_dms(lon2_deg, lon2_min, lon2_sec),
+                                                      join_dms(web_dir_ra_deg, web_dir_ra_min, web_dir_ra_sec))
+
+        dir_lat2, dir_lon2, dir_az2to1 = vincdir(dms2dd(lat1), dms2dd(lon1), inv_az1to2, inv_ell_dist)
+
+        np.testing.assert_almost_equal((float(web_dir_lat2), float(web_dir_lon2), float(web_dir_az2to1)),
+                                       (dd2dms(dir_lat2), dd2dms(dir_lon2), dd2dms(dir_az2to1)),
+                                       decimal=6,
```

Comment:
The decimal rounding assertion for lat and long should be around 8 decimal places, azimuth 2 to 1 is fine at 6 decimal places.